MeetupPlatform.Api/Features/Meetups/LeftRootComment/LeftRootCommentFeature.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace MeetupPlatform.Api.Features.Meetups.LeftRootComment;
+
+using AutoMapper;
+using MeetupPlatform.Api.Authentication.Helpers;
+using MeetupPlatform.Api.Domain.Comments;
+using MeetupPlatform.Api.Persistence.Context;
+using MeetupPlatform.Api.Seedwork.WebApi;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Meetups)]
+public class LeftRootCommentFeature : FeatureBase
+{
+    private readonly ApplicationContext context;
+    private readonly IMapper mapper;
+
+    public LeftRootCommentFeature(ApplicationContext context, IMapper mapper)
```

Comment:
User (or API consumer) should not be exposed to such concepts like `root` comment. Instead we should use more real-world names, like `LeftMeetupComment` or `ReplyToComment` (even if it's basically the same thing).